centralledger/values.yaml

```diff
@@ -30,7 +30,7 @@ centralledger-service:
     api:
       image:
         repository: mojaloop/central-ledger
-        tag: v6.1.0
+        tag: v6.1.1
```

Comment:
this should be 6.1.0

Reply:
Updated - version: 6.1.0

Reply:
Not sure about this one. all other tag: are currently 6.1.1 related to central-ledger.